<?xml version="1.0"?>

<!--
Launch file to test the maskor_thermal_mapper_node.

You need a bagfile or real xtion and thermal image stream
-->
<launch>
 
 <node pkg="rosbag" type="play" name="rosbag" args="-l [PATH TO BAGFILE WITH DEPTH AND THERMAL IMAGES].bag"/> 

 <!-- launch mapping node-->
 <node 
     name="maskor_thermal_mapper_node"
     pkg="maskor_thermal_mapper" 
     type="maskor_thermal_mapper_node" 
     respawn="false" 
     launch-prefix="gnome-terminal -e" 
     >
     <param name="package_name" value="maskor_thermal_mapper"/>
     <param name="database_path" value="/database"/>
     <param name="yaml_path" value="/full_flir_ir_stereo_calib.yaml"/>
     <param name="publish_topic" value="flir_camera/mapped/image"/>
     <param name="enable_colormap" value="true"/>
 </node>
   
</launch>
